src/main/java/racingcar/domain/Cars.java

```diff
@@ -2,41 +2,54 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
 
 public class Cars {
-    private final List<Car> carInformation;
-    private final List<String> winner = new ArrayList<>();
+    private static final int RANDOM_NUM_MAX_VALUE = 10;
+    private static final Random random = new Random();
+    private final List<Car> cars;
 
-    public Cars(List<Car> carInformation) {
-        this.carInformation = carInformation;
+    public Cars(List<String> carNames) {
+        this.cars = convertToCar(carNames);
     }
 
-    public void addCarInformation(Car car) {
-        this.carInformation.add(car);
+    private List<Car> convertToCar(List<String> carNames) {
+        return carNames.stream().map(Car::new).collect(Collectors.toList());
     }
 
-    public List<Car> getCarInformation() {
-        return this.carInformation;
+    public List<Car> runRound() {
+        for (Car car : this.cars) {
+            int randomNumber = random.nextInt(RANDOM_NUM_MAX_VALUE);
+            car.runForward(randomNumber);
+        }
+        return this.cars;
     }
 
-    public int findMaxDistance() {
+    public List<String> getWinner() {
+        List<String> winner = new ArrayList<>();
+        int maxDistance = findMaxDistance();
+        for (Car car : this.cars) {
+            compareDistance(winner, car, maxDistance);
+        }
+        return winner;
+    }
+
+    private int findMaxDistance() {
         int maxDistance = -1;
-        for (Car car : this.carInformation) {
-            maxDistance = Math.max(car.getDistance(), maxDistance);
+        for (Car car : this.cars) {
+            maxDistance = Math.max(car.getDistance().getDistance(), maxDistance);
```

Comment:
stream 의 max 를 사용해볼수도 있겠어요!

```suggestion
    return something.stream()
                .mapToInt(it -> it.getValue())
                .max()
                .getAsInt();
```

Reply:
이전 리팩토링 과정에서 For문을 stream으로 바꾸며, 해당 코드도 stream을 사용할 수 있을까 라는 생각을 해보았는데 어떤 중간연산자를 사용해야 할 지 고민이었는데 코멘트를 보고 stream max 에 대해 찾아보며 적용해 볼 수 있었습니다 !
```
    private int findMaxDistance() {
        return this.cars.stream()
                .mapToInt(car -> car.getDistance().getValue())
                .max()
                .orElse(-1);
    }
```

Reply:
OptionalInt 를 잘 핸들링해주셨군요!
Optional 사용방법에 대해서 추가로 아티클 공유해드릴게요 😃 

http://homoefficio.github.io/2019/10/03/Java-Optional-%EB%B0%94%EB%A5%B4%EA%B2%8C-%EC%93%B0%EA%B8%B0/